blog/2023-06-30-app-competition-results.md

```diff
@@ -0,0 +1,65 @@
+---
+slug: 2023/06/app-competition-results
+title: 2023 App Competition Results
+authors: rene
+tags: [dac2023, app-competition, android, web, apps]
+---
+
+*As we’ve recently had the DHIS2 Annual Conference, we’ve also completed the 2023 App Competition tied to the conference. On Thursday, the finalists were able, in 7 minutes, to present their Android or Web application, and once all were done, the audience decided which app they thought was the best.*
+
+<!--truncate-->
+
+But the competition didn’t start during the conference. Several months ago, the app competition for 2023 was announced, and the community was able to submit the application they thought would be able to impress the audience and the jury. But impressing the audience is one thing, actually having a useful application that could be used by more than just those who’ve developed it, was much more important. And all of the submissions met that criteria. 
+
+The DHIS2 community is more than just a few pockets of local organizations. We all face similar (or identical) problems, diseases, and challenges. And because of this, developing apps that can be used all over the globe is an incredibly good use of the developers’ time. This is why this was one of the important factors when judging the application, can it be reused by other organisations.
+
+ The submissions this year were of a very high quality, and the jury had a hard time choosing which app should be a finalist. They would’ve preferred to have many more finalists as many of the submissions were of such a high quality, but after careful consideration five applications were chosen to be the finalists, which were as followed:
+
+- **DHIS2 Visualisation Studio**
+    - Developer: Albert Mutesasira
+    - Organisation: HISP Uganda
+- **JumpaDokter**
+    - Developer: Saldi Yusuf
+    - Organisation: HISP Indonesia
+- **DHIS2 Analytics Messenger**
+    - Developer: Eric Chingalo
+    - Organisation: HISP Tanzania
+- **Growth and Nutrition Monitoring System**
+    - Developer: Chaturanga Manchanayake
+    - Organisation: HISP Sri Lanka
+- **Homepage App**
+    - Developer: Ignacio Foche Pérez
+    - Organisation: EyeSeeTea
+    
+You can read more about these applications in the finalists page on the Community of Practice here: https://community.dhis2.org/t/2023-dhis2-app-competition-finalists/53264
+    
+We’ve had a diverse field of finalists, two of the applications are built for Android and three for the web. All of the web applications were developed to be installed via the App Hub, while the Android apps have no such luxury of an integrated app hub, they also both are able to be deployed by other organisations as they are, or will be, open sourced.
```

Comment:
I think the second sentence should be reformulated, particularly the part "while the Android apps have no such luxury of an integrated app hub..." as at the moment it kind of reads as a critique that we do not have Android apps available in the app hub? 

Why not just make the sentence state where users can find and download the apps?